README.md

```diff
@@ -1,43 +1,152 @@
-## mcmicro-nf: Nextflow prototype for mcmicro
+# mcmicro-nf: Multiple-choice microscopy pipeline
+
+## Installation
 
 Install [Nextflow](https://www.nextflow.io/): `curl -s https://get.nextflow.io | bash`
 
-### Local execution
+### Additional steps for local installation
+* Install [Docker](https://docs.docker.com/install/). Ensure that the Docker engine is running by typing `docker images`. If the engine is running, it should return a (possibly empty) list of container images currently downloaded to your system.
+* (Optional) If working with TMAs, you will need MATLAB 2018a or later. Additionally, you will need to install Coreograph locally by running `nextflow run labsyspharm/mcmicro-nf/setup.nf`.
+
+## Exemplar data
+
+Two exemplars are currently available for demonstration purposes. These are
+
+* `exemplar-001` is meant to serve as a minimal reproducbile example for running all modules of the pipeline, except the dearray step. The exemplar consists of six tile images, collected on a Lung Adenocarcinoma specimen in three cycles, for a total of 12 channels. Because the exemplar is small, illumination profiles were precomputed from the larger context and included with the raw images.
```

Comment:
* typo: reproduc**ib**le
* lowercase lung adenocarcinoma
* 10 cycles, not 3
* mention that tiles are each 4-channel.

Possibly just rewrite second sentence: "The exemplar consists of a small lung adenocarcinoma specimen taken from a larger TMA (tissue microarray), imaged using CyCIF with ten cycles. Each cycle consists of six four-channel image tiles for a total of forty channels."

Reply:
While 10 cycles are available on S3. The download script only pulls the first three of them, per this discussion: https://github.com/labsyspharm/mcmicro-nf/issues/15